README.md

```diff
@@ -97,6 +97,10 @@ And you can run that simply as:
 
     go run main.go
 
+> [!NOTE]  
+> The first compilation of Fyne on Windows _can_ take up to 10 minutes, depending on your hardware. Subsequent builds will be fast.
+This is caused because the C compiler on Windows lacks optimizations present on Linux and MacOS.
```

Comment:
I'm not sure we want to say what the cause is. We're not 100% certain that it's due to the C compiler (it seems Windows Defender may also play a role?) and even so we might not want to "bad mouth" the Windows C compiler ;)

Reply:
Defender doesn't really do much. I have it uninstalled and it is still slow on the first compile.